python/main.py

```diff
@@ -18,14 +24,47 @@
     allow_headers=["*"],
 )
 
+
+def add_json(data,file_json='item.json'):
+    path = Path(file_json)
+    if not path.is_file():
+        file_data={"items": []}
+    else:
+        with open(file_json,'r')as f:
+            file_data=json.load(f)
+
+
+    with open(file_json, 'w') as f:
+        file_data["items"].append(data)
+        json.dump(file_data, f)
+    print (file_data)
+
+
 @app.get("/")
 def root():
     return {"message": "Hello, world!"}
 
+
 @app.post("/items")
-def add_item(name: str = Form(...)):
-    logger.info(f"Receive item: {name}")
-    return {"message": f"item received: {name}"}
+def add_item(name: str = Form(...), category: str = Form(...)):
+    add_json({"name": name, "category": category})
+    logger.info(f"Receive item: {name} , {category}")
+    return {"message": f"item received: {name} , {category}"}
+
+@app.get("/items")
+def get_item():
+    with open("item.json",'r')as f:
+        file_data=json.load(f)
+    return file_data
+
+@app.delete("/items")
+def init_item(file_json='item.json'):
+    path = Path(file_json)
+    if os. path. exists(path):
```

Comment:
```suggestion
    if os.path.exists(path):
```
このままの状態だと動かない気がするんですが、動いてますか？

Reply:
一応動きますが、修正します。ありがとうございます！